backend/Services/Auth/Adapters/LDAP.php

```diff
@@ -52,6 +53,7 @@ public function init(array $config = [])
                 $this->ldap_bindPass = $config['ldap_bindPass'];
                 $this->ldap_baseDN = $config['ldap_baseDN'];
                 $this->ldap_filter = $config['ldap_filter'];
+                $this->ldap_attributes = $config['ldap_attributes'];
```

Comment:
Can you avoid a braking change here please?
```suggestion
                $this->ldap_attributes = isset($config['ldap_attributes']) ? $config['ldap_attributes'] : ['*'];
```